scripts/get-next-version.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-console */
+
+const fail = (...reason) => {
+  console.error(...reason)
+  process.exit(1)
+}
+
+const bump = require('conventional-recommended-bump')
+const currentVersion = require('../package.json').version
+
+bump({ preset: 'angular' }, (err, { releaseType }) => {
+  if (err) {
+    return fail('Error getting next version', err)
+  }
+
+  let [major, minor, patch] = currentVersion.split('.').map(Number)
```

Comment:
Could use [`semver.inc`](https://github.com/npm/node-semver#functions) instead here. Looks like we already have it as a dependency.

Reply:
nice, updated